https/README.md

```diff
@@ -0,0 +1,10 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains files and a template config for the implementation of TLS.
```

Comment:
Suggestion: "The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP."